packages/dashboard-v2/src/components/forms/LoginForm.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from "prop-types";
+import { Formik, Form } from "formik";
+import { Link } from "gatsby";
+import * as Yup from "yup";
+
+import HighlightedLink from "../HighlightedLink";
+import { TextField } from "../Form/TextField";
+import { Button } from "../Button";
+
+import accountsService from "../../services/accountsService";
+
+const loginSchema = Yup.object().shape({
+  email: Yup.string().required("Email is required").email("Please provide a valid email address"),
+  password: Yup.string().required("Password is required"),
+});
+
+const INVALID_CREDENTIALS_ERRORS = ["password mismatch", "user not found"];
+
+export const LoginForm = ({ onSuccess }) => (
+  <Formik
+    initialValues={{
+      email: "",
+      password: "",
+    }}
+    validationSchema={loginSchema}
+    onSubmit={async (values, { setErrors }) => {
+      try {
+        await accountsService.post("login", {
+          json: values,
+        });
+
+        onSuccess();
+      } catch (err) {
+        if (err.response) {
+          const data = await err.response.json();
+
+          if (INVALID_CREDENTIALS_ERRORS.includes(data.message)) {
+            setErrors({
+              email: "Invalid e-mail address or password",
```

Comment:
we should use consistent spelling, should be email

Reply:
Fixed 👍